lib/config/definitions.ts

```diff
@@ -251,6 +251,13 @@ const options: RenovateOptions[] = [
     type: 'boolean',
     default: false,
   },
+  {
+    name: 'dockerChildPrefix',
+    description:
+      'Change this value in order to add a prefix to the Renovate Docker sidecar image names and labels.',
+    type: 'string',
+    admin: true,
```

Comment:
```suggestion
    admin: true,
    default: 'renovate_',
```

Set default, so it's visible in docs.